### DISCUSSION TITLE
SwinIR config settings for GAN training

### DISCUSSION THREAD

**Discussion by PeterVennerstrom** [open-mmlab/mmediting#1852] [How-to] [answered]
The [GAN configs](https://github.com/open-mmlab/mmagic/blob/main/configs/swinir/swinir_gan-x2s64w8d6e180_8xb4-lr1e-4-600k_df2k-ost.py) for SwinIR Real World SR appear to only change the experiment name after inheriting the psnr settings.

```
_base_ = ['swinir_psnr-x2s64w8d6e180_8xb4-lr1e-4-600k_df2k-ost.py']

experiment_name = 'swinir_gan-x2s64w8d6e180_8xb4-lr1e-4-600k_df2k-ost.py'
work_dir = f'./work_dirs/{experiment_name}'
save_dir = './work_dirs/'
```

I haven't found any settings related to the GAN model definition or loss.  The setting to load the psnr trained model for fine-tuning with GAN loss is also not seen in the child config.

Any help locating these settings would be appreciated!

**Comment by Z-Fran** [comment, marked as answer]
We only support the inference of swinir and inference only needs `generator` of GAN, so two configs are the same. If you want to train the model, you can use the official codebase of swinir from  https://github.com/JingyunLiang/SwinIR/tree/main and https://github.com/cszn/KAIR/tree/master .

**Reply by PeterVennerstrom** [reply]
Thank you for your quick response.
